apps/api/src/domain/models/term/queries/term-query-repository.interface.ts

```diff
@@ -17,13 +19,24 @@ export interface AudioCandidatesForTerm {
     possibleAudioItems: EventSourcedAudioItemViewModel[];
 }
 
+interface PaginatedResponse<T> {
+    entities: T[];
+    page: number;
+    count: number;
```

Comment:
Do we want to call this count? What if we want to know the total, unfiltered size of a collection as well?
- available
- numberOfMatches
- count
- counts: { filtered: number, total: number }
- numberAvailable

Reply:
If we keep it the same, we should add a detailed comment.